src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -39,22 +43,97 @@ internal NativeIntegerTypeSymbol(NamedTypeSymbol underlyingType) : base(underlyi
+        ///   sizeof() should be used instead of Size;
+        ///   + and - should be used instead of Add() and Subtract();
+        ///   ToInt32(), ToInt64(), ToPointer() should be used from underlying types only.
+        /// The remaining members are exposed on the native integer types with appropriate
+        /// substitution of underlying types in the signatures.
         /// </summary>
-        public override ImmutableArray<Symbol> GetMembers() => ImmutableArray<Symbol>.Empty;
+        public override ImmutableArray<Symbol> GetMembers()
+        {
+            if (_lazyMembers.IsDefault)
+            {
+                ImmutableInterlocked.InterlockedInitialize(ref _lazyMembers, makeMembers(_underlyingType.GetMembers()));
+            }
+            return _lazyMembers;
+
+            ImmutableArray<Symbol> makeMembers(ImmutableArray<Symbol> underlyingMembers)
+            {
+                var builder = ArrayBuilder<Symbol>.GetInstance();
+                builder.Add(new SynthesizedInstanceConstructor(this));
+                foreach (var underlyingMember in underlyingMembers)
+                {
+                    Debug.Assert(_underlyingType.Equals(underlyingMember.ContainingSymbol));
+                    switch (underlyingMember)
+                    {
+                        case MethodSymbol underlyingMethod:
+                            if (underlyingMethod.IsGenericMethod)
+                            {
+                                break;
+                            }
+                            switch (underlyingMethod.MethodKind)
+                            {
+                                case MethodKind.Ordinary:
+                                    if (underlyingMethod.IsOverride)
+                                    {
+                                        addMethodIfAny(builder, underlyingMethod);
+                                    }
+                                    else
+                                    {
+                                        switch (underlyingMethod.Name)
+                                        {
+                                            case "Add":
+                                            case "Subtract":
+                                            case "ToInt32":
+                                            case "ToInt64":
+                                            case "ToUInt32":
+                                            case "ToUInt64":
+                                            case "ToPointer":
+                                                break;
+                                            default:
+                                                addMethodIfAny(builder, underlyingMethod);
+                                                break;
+                                        }
+                                    }
+                                    break;
+                                case MethodKind.ExplicitInterfaceImplementation:
```

Comment:
>MethodKind.ExplicitInterfaceImplementation [](start = 37, length = 42)

I do not think we can trust this completely. I.e. I think it is possible to have a different kind and still be explicit interface implementation. What exactly are we trying to accomplish here? Perhaps we should simply add everything that is MetadataVirtual, that would take care of all overrides and interface implementations.